src/java.desktop/share/classes/javax/swing/ImageIcon.java

```diff
@@ -61,6 +61,11 @@
  * from Images. Images that are created from a URL, filename or byte array
  * are preloaded using MediaTracker to monitor the loaded state
  * of the image.
+ * If the image source parameter to a constructor or method is non-null,
+ * but does not reference valid accessible image data,
+ * no exceptions will be thrown but the image will be unset,
+ * as it will have no dimensions and never be drawn, and
+ * {@code getImageLoadStatus()} will report {@see java.awt.MediaTracker#ERRORED}.
```

Comment:
```suggestion
 * {@code getImageLoadStatus()} will report {@link java.awt.MediaTracker#ERRORED}.
```
You shouldn't use `@see` here, it should either be `@code` or `@link`. I think both references should use the same tag: either `@code` or `@link`; the latter allows quickly navigating to the *method* or constant.

Perhaps, `@link` for the `getImageLoadStatus` method would work better: it allows viewing the description of the method and its purpose; in its turn, `getImageLoadStatus` references possible return values including `MediaTracker.ERRORED` with `@see` tag.

Reply:
Indeed, this is not supported, the generated javadoc looks like this:

“…and `getImageLoadStatus()` will report .”

Therefore I suggest the following:

```java
 * {@link #getImageLoadStatus()} will report {@code MediaTracker.ERRORED}.
```